Lib/logging/config.py

```diff
@@ -68,8 +68,9 @@ def fileConfig(fname, defaults=None, disable_existing_loggers=True, encoding=Non
             cp.read_file(fname)
         else:
             encoding = io.text_encoding(encoding)
-            cp.read(fname, encoding=encoding)
-
+            files = cp.read(fname, encoding=encoding)
```

Comment:
This can be kept in case e.g. there's an invalid file there, but it would IMO be better to check for file existence and bail with a `FileNotFoundError` if there's no such file. That's more informative. Then this error (no parsed filenames) would lead to a message "The configuration file XXX appears not to be in the expected format" or some such.